substrate/client/executor/common/src/runtime_blob/runtime_blob.rs

```diff
@@ -192,7 +193,7 @@ impl RuntimeBlob {
 		match self.0 {
 			BlobKind::WebAssembly(raw_module) =>
 				serialize(raw_module).expect("serializing into a vec should succeed; qed"),
-			BlobKind::PolkaVM(ref blob) => blob.as_bytes().to_vec(),
+			BlobKind::PolkaVM(ref blob) => blob.1.to_vec(),
```

Comment:
And here's a good example of what I meant in a comment above. In this occurrence, it's `PolkaVM(ref blob)`, and in the next one, it's `PolkaVM((ref blob, _))`. That is, we've already started to call two different things a "blob". That's error-prone and causes confusion, and overall, there's a lot of room for design improvement here.